src/lib/style/components/entity_finder/_details-view.scss

```diff
@@ -2,6 +2,7 @@
 @use './entity-finder' as EntityFinder;
 
 .DetailsViewReflexElement {
+  right: $-splitter-width;
```

Comment:
See splitter-widening [here](https://github.com/Sage-Bionetworks/Synapse-React-Client/pull/981/files#r616020656). Moving the table here makes the splitter seem less wide while still having a wider click area